pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUmbrellaStudyCached.java

```diff
@@ -139,7 +139,7 @@ public int updateShareLocationForStudy(boolean shareLocation, String guid) {
         if (val > 0) {
             StudyDto dto = findByStudyGuid(guid);
             if (dto != null) {
-                notifyModelUpdated(ModelChangeType.UMBRELLA, dto.getId());
+                notifyModelUpdated(ModelChangeType.STUDY, dto.getId());
```

Comment:
This "thing" that's changing here is the study. STUDY and UMBRELLA are totally different things. I don't know what "notify model updated" really means, but I think this should say STUDY instead.

Reply:
I think you are right to change it to STUDY.

Idea with the notifyModelUpdated is to alert other caches that a change has occurred that might have made their cached data invalid.

We had currently defined 3 scopes for model changes: user, study, and umbrella. As names suggest, they refer to data that is just user-specific, or study specific or for a whole umbrella of studies.
If our data was perfectly factored/normalized then each cache would have data that is not replicated anywhere else. But our data model has data that is replicated in multiple spots (e.g., activity instances have some data that is also in activities and activity summaries, etc).

So invention of the change scopes was let other caches that share that scope that they should invalidate themselves.

In this case, the a change to the study should only affect stuff in the study scope (despite having "umbrella" as part of this name which is probably why messed up here).